app/client/src/reducers/entityReducers/pageListReducer.tsx

```diff
@@ -20,6 +20,10 @@ import type { Page } from "entities/Page";
 const initialState: PageListReduxState = {
   pages: [],
   isGeneratingTemplatePage: false,
+  isGeneratePageModalOpen: {
+    value: false,
+    params: {},
+  },
```

Comment:
```suggestion
  generatePage: {
    modalOpen: false,
    params: {},
  },
```

this would be a better name here